builtin/k8s/deployment.go

```diff
@@ -24,13 +24,15 @@ func (d *Deployment) newDeployment(name string) *appsv1.Deployment {
 		Spec: appsv1.DeploymentSpec{
 			Selector: &metav1.LabelSelector{
 				MatchLabels: map[string]string{
+					"app":  name,
 					"name": name,
 				},
 			},
 
 			Template: corev1.PodTemplateSpec{
 				ObjectMeta: metav1.ObjectMeta{
 					Labels: map[string]string{
+						"app":  name,
 						"name": name,
```

Comment:
Not sure where this is happening exactly, but it seems like we use `name` some where for the local release URL to resolve so maybe we leave this as is and include app too?

Reply:
That's a reasonable decision. I would add a comment to that effect for future-us.